### PR TITLE
[PRP-340] Changing label elements, correcting screen reader behavior.…

### PR DESCRIPTION
**Pull request by microwavenby**
… Updating i18n text

## Ticket

https://wicmtdp.atlassian.net/browse/PRP-340

## Changes
* Document item is no longer a `<Label>` element
* `<Label>` element is the text in the drop target "Click to choose a file"
* I18n strings changed to indicate "Click" behavior

## Context for reviewers
* We CAN disable the file input component's focus and  setaria-hidden and use the label's click through behavior to trigger the browser's picker -- but it means coding some custom javascript to ensure the space / enter keys work when tabbing through page elements. See [this page](https://accessabilly.com/a-styled-accessible-file-upload/) for a bit of info on the kind of work we're discussing

## Testing
1.  Go through the page flow to `/upload`
2. Turn on your screen reader
3. Tab through the page to the file picker
4. Hear (or see with visual cues in the screen reader) "Click to choose a file to upload", of type "button"

upload-accessibility

This page currently passes Lighthouse accessibility as well

### CODE REVIEW

File: participant/app/components/FileUploader.tsx [hunk 1]
```diff
@@ -200,7 +200,7 @@ export const FileUploaderForwardRef: React.ForwardRefRenderFunction<
     <FormGroup>
       {children}
       {previews}
-      <Label htmlFor={id}>{currentDocumentString}</Label>
+      <div className="margin-top-3">{currentDocumentString}</div>
```
Comment (microwavenby) [inline participant/app/components/FileUploader.tsx:203]: This is the spacing of the USA Label element; should keep visual alignment the same.

File: participant/styles/_uswds-theme-custom-styles.scss [hunk 2]
```diff
@@ -54,6 +54,10 @@ p {
   pointer-events: auto;
 }
 
+.usa-label {
+  pointer-events: auto;
+}
+
```
Comment (microwavenby) [inline participant/styles/_uswds-theme-custom-styles.scss:60]: Labels should be clickable. It's wild that USWDS styles them otherwise.

**Review by rocketnova — APPROVED** [review]
LGTM! I'm deploying this branch to `dev` so that @danielladevera-nava can run through the screen reader test. I'll wait to merge until after that review

**Comment by danielladevera-nava** [comment]
Went through the dev build and validated VoiceOver read this as expected per the testing instructions. TYSM!! 🥇

**Review by danielladevera-nava — COMMENTED** [review]
validated change via VoiceOver on dev

**Pull request merged**